proof/access-control/ARM/ArchSyscall_AC.thy

```diff
@@ -179,7 +179,7 @@ global_interpretation Syscall_AC_1?: Syscall_AC_1
 proof goal_cases
   interpret Arch .
   case 1 show ?case
-    by (unfold_locales; (fact Syscall_AC_assms)?)
+    by (unfold_locales; wpsimp wp: Syscall_AC_assms)
```

Comment:
Do we need wpsimp for this one? I think we discussed this before, and I'm starting to lean towards keeping interfaces more strictly. The point of the `Syscall_AC_assms` set is that it should collect the interface statements -- so if there is still proof work necessary at this point, then something is not quite right.